locales/pt/json.json

```diff
@@ -114,18 +114,18 @@
     "Benin": "Benim",
     "Bermuda": "Bermudas",
     "Bhutan": "Butão",
-    "Billing Information": "Informação de facturação",
-    "Billing Management": "Gestão de facturação",
+    "Billing Information": "Informação de faturação",
+    "Billing Management": "Gestão de faturação",
     "Bolivia": "Bolívia",
     "Bolivia, Plurinational State of": "Estado plurinacional da Bolívia",
-    "Bonaire, Sint Eustatius and Saba": "Bonaire, Sint Eustatius and Sábado",
-    "Bosnia And Herzegovina": "Bósnia E Herzegovina",
+    "Bonaire, Sint Eustatius and Saba": "Bonaire, Santo Eustáquio e Saba",
+    "Bosnia And Herzegovina": "Bósnia And Herzegovina",
     "Bosnia and Herzegovina": "Bósnia e Herzegovina",
```

Comment:
I think you need to update here too as per the option above.

Reply:
The original spelling of “And” and “and” is because these phrases are used in different Laravel packages and the spelling is not always the same.

What is the correct spelling in Portuguese - “Bósnia E Herzegovina” or “Bósnia e Herzegovina”?

I mean that it is better to specify the same correct translation in both phrases as in [Spanish](https://github.com/Laravel-Lang/lang/blob/main/locales/es/json.json#L122-L123), for example.

Reply:
Totally agree, I used "E" because the "And" was uppercase, but it's way better to normalize and just use "e" when we have "And" or "and".